timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/database/README.md

```diff
@@ -1,6 +1,3 @@
-# DataAccess
-> Wrapper around data access methods
-
```

Comment:
I think there's more in this documentation that needs fixing (the class DataAccess is mentioned here and there)